packages/configuration-development/index.json

```diff
@@ -0,0 +1,36 @@
+{
+    "connector": "local",
+    "formatters": [
+        "html",
+        "summary"
+    ],
+    "hints": {
+        "axe": "error",
+        "babel-config/is-valid": "error",
+        "disown-opener": "error",
+        "highest-available-document-mode": "error",
+        "meta-charset-utf-8": "error",
+        "meta-viewport": "error",
+        "no-bom": "error",
+        "no-protocol-relative-urls": "error",
+        "sri": "error",
```

Comment:
Does SRI work? Especially if the asset is local

Reply:
Per our offline discussion, this won't work since we don't fetch the linked resources in the local connector. I'll update the PR to remove `hint-sri` and file an issue to track enhancing it to run in this context in the future.